### PR TITLE
fix: Bring valid selection into multi input component

### PR DESCRIPTION
**Pull request by JKMarkowski**
#### Please provide a link to the associated issue.
part of https://github.com/SAP/fundamental-ngx/issues/3085
#### Please provide a brief summary of this pull request.
Now multi input uses same structure as list with selection. Temporary there are added some styles, to keep fiori3 design. 
Also there is fixed focusing/selecting items by keyboard
#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/master/CONTRIBUTING.md
- [x] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist

### CODE REVIEW

File: libs/core/src/lib/list/list-item/list-item.component.ts [hunk 1]
```diff
@@ -68,6 +68,10 @@ export class ListItemComponent implements KeyboardSupportItemInterface, AfterCon
     @HostBinding('class.fd-list__item--link')
     link = false;
 
+    /** Defines if list item is used inside MultiInputComponent */
```
Comment (mikerodonnell89) [inline libs/core/src/lib/list/list-item/list-item.component.ts:71]: At the very least this should be hidden from the app developer, but I think we do not need to use an input at all, the multi-input could get list item content children and set the property there.  Or this list item could ascend a few parents and figure out if it is in a multi input.

**Review by mikerodonnell89 — CHANGES_REQUESTED** [review]
Looks good other than this one thing

File: libs/core/src/lib/list/list-item/list-item.component.ts (hunk 1 above)
Comment (JKMarkowski) [reply libs/core/src/lib/list/list-item/list-item.component.ts]: @mikerodonnell89 You are right, it should be controlled from parent. Anyway it has been changed, so now it uses selection list for multi input with some minor styles adjustments.

**Review by salarenko — APPROVED** [review]

**Review by mikerodonnell89 — CHANGES_REQUESTED** [review]
I think after opening the popover, user should be able to tab in to the list.  Currently I have to click a list item to focus it before I can use the arrow keys to nav through the list

**Review by mikerodonnell89 — APPROVED** [review]

**Review by InnaAtanasova — APPROVED** [review]

**Pull request merged**